src/website/app/demos/Table/bestPractices.js

```diff
@@ -3,6 +3,47 @@ import React from 'react';
 import Table from '../../../../library/Table';
 
 export default [
+  {
+    type: 'do',
+    description: `If one or more columns in your table are sortable, make sure
+one of those columns has a default sort applied, so that its clear the table can
+be sorted.`,
+    example: (
+      <Table
+        data={[
+          { product: 'Fruit Basket', price: '$123.45' },
+          { product: 'Muffin Basket', price: '$98.76' }
+        ]}
+        columns={[
+          { key: 'product', content: 'Product', primary: true },
+          { key: 'price', content: 'Price', textAlign: 'end' }
+        ]}
+        sortable
+        defaultSort={{ key: 'product' }}
+        title="Example data"
+        hideTitle
+      />
+    )
+  },
+  {
+    type: 'do',
+    description: `Designate a column as \`primary\` in your column definitions,
+unless your data has no column to serve that purpose.`,
+    example: (
+      <Table
+        data={[
+          { product: 'Fruit Basket', price: '$123.45' },
+          { product: 'Muffin Basket', price: '$98.76' }
+        ]}
+        columns={[
+          { key: 'product', content: 'Product', primary: true },
+          { key: 'price', content: 'Price', textAlign: 'end' }
+        ]}
+        title="Example data"
+        hideTitle
+      />
+    )
+  },
```

Comment:
During Pagination research, I came across a couple other best practices we may want to mix in here: 
* Along with our number-alignment example, maybe add a note that if using decimals, be consistent with decimal places 
* Avoid adding units in data cells; include in header instead
* Don't use Table for data sets with a blend of text, images, and data visualizations, or content with mixed formatting; use Card instead

Reply:
^^ if we want to use that 3rd point, it should be integrated into the when/how to use instead